<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>623</width>
    <height>344</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="enrollPage">
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>180</x>
          <y>30</y>
          <width>291</width>
          <height>221</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="welcomeLabel">
           <property name="text">
            <string>Welcome Soldier</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="enrollButton">
           <property name="text">
            <string>PushButton</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="namePage">
       <widget class="QPushButton" name="nextNamePageButton">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>230</y>
          <width>87</width>
          <height>26</height>
         </rect>
        </property>
        <property name="text">
         <string>Next</string>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>140</x>
          <y>87</y>
          <width>321</width>
          <height>61</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="nameInputLabel">
           <property name="text">
            <string>Name : </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="nameInputBox"/>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="avatarSelectPage">
       <widget class="QPushButton" name="avatar1">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>87</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">
image: url(:/images/avatar_pics/test1.jpeg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="avatar2">
        <property name="geometry">
         <rect>
          <x>260</x>
          <y>80</y>
          <width>87</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/images/avatar_pics/test2.jpeg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="avatar3">
        <property name="geometry">
         <rect>
          <x>420</x>
          <y>80</y>
          <width>87</width>
          <height>91</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/images/avatar_pics/test3.jpeg);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="displayPage">
       <widget class="QLabel" name="nameDisplayLabel">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>110</y>
          <width>251</width>
          <height>18</height>
         </rect>
        </property>
        <property name="text">
         <string>Your Name</string>
        </property>
       </widget>
       <widget class="QPushButton" name="AvatarDisplayButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>90</x>
          <y>80</y>
          <width>87</width>
          <height>81</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>623</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
